lncfg/address_test.go

```diff
@@ -265,3 +266,56 @@ func validateAddr(t *testing.T, addr net.Addr, test addressTest) {
 		)
 	}
 }
+
+func TestIsPrivate(t *testing.T) {
+	nonPrivateIPList := []net.IP{
+		net.IPv4(169, 255, 0, 0),
+		{0xfe, 0x79, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
+		net.IPv4(225, 0, 0, 0),
+		{0xff, 0x01, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
+		net.IPv4(11, 0, 0, 0),
+		net.IPv4(172, 15, 0, 0),
+		net.IPv4(192, 169, 0, 0),
+		{0xfe, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
+		net.IPv4(8, 8, 8, 8),
+		{2, 0, 0, 1, 4, 8, 6, 0, 4, 8, 6, 0, 8, 8, 8, 8},
+	}
+	privateIPList := []net.IP{
+		net.IPv4(169, 254, 0, 0),
+		{0xfe, 0x80, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
+		net.IPv4(224, 0, 0, 0),
+		{0xff, 0x02, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
+		net.IPv4(10, 0, 0, 1),
+		net.IPv4(172, 16, 0, 1),
```

Comment:
Maybe add the upper bound of B class (`172.31.255.255`) as well?

Reply:
sure, added